mpcstats_lib.py

```diff
@@ -213,33 +228,59 @@ def _(i):
 
     return highest
 
-# LATER
+
+def variance(data: list[sint]):
+    return _variance(data, True)
+
+
+def linear_regression(xs: list[sint], ys: list[sint]):
```

Comment:
Is there a reference implementation for linear regression? If so, could we leave the URL for future reference?
```suggestion
def linear_regression(xs: list[sint], ys: list[sint]):
    # Ref: URL
```

Reply:
I used the formula in the Simple linear regression wiki page: https://en.wikipedia.org/wiki/Simple_linear_regression. I added a comment with the URL.